kd_sweep.sh

```
@@ -260,47 +260,49 @@ elif [[ "$MODE" == "run_all_4m" ]]; then
 
   # 13) SampledKD LinUCB k=25
-  sbatch train.slurm linucb 25 light "$KD_SWEEP_TAG"
+  sbatch --wait train.slurm linucb 25 light "$KD_SWEEP_TAG"
 
   # 10) SampledKD Score k=25 (enable score-based selection)
-  sbatch --export=ALL,SCORE_TOKEN_SELECTION=1 train.slurm top-k-tok 25 light "$KD_SWEEP_TAG"
+  sbatch --wait --export=ALL,SCORE_TOKEN_SELECTION=1 train.slurm top-k-tok 25 light "$KD_SWEEP_TAG"
+
+  # # 12) SampledKD Trained also on GSM8K k=25 (GSM8K-only run)
+  # # Note: train.slurm supports DATASETS / DATASET_CONFIG / PROMPT_COL / ANSWER_COL overrides
+  sbatch --wait --export=ALL,DATASETS="gsm8k",DATASET_CONFIG="main",PROMPT_COL="question",ANSWER_COL="answer" \
```

Shellcheck (0.11.0)

[warning] Word is of the form "A"B"C" (B indicated). Did you mean "ABC" or "A\"B\"C"?

(SC2140)